queue_job_batch/README.rst

```diff
@@ -0,0 +1,117 @@
+===============
+Job Queue Batch
+===============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fqueue-lightgray.png?logo=github
+    :target: https://github.com/OCA/queue/tree/11.0/queue_job_batch
+    :alt: OCA/queue
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/queue-11-0/queue-11-0-queue_job_batch
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/230/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This addon adds an a grouper for queue jobs.
+
+It allows to show your jobs in a batched form in order to know better the
+results.
+
+Example:
+
+.. code-block:: python
+
+  from odoo import models, fields, api
+  from odoo.addons.queue_job.job import job
+
+  class MyModel(models.Model):
+     _name = 'my.model'
+
+     @api.multi
+     @job
+     def my_method(self, a, k=None):
+         _logger.info('executed with a: %s and k: %s', a, k)
+
+
+  class MyOtherModel(models.Model):
+      _name = 'my.other.model'
+
+      @api.multi
+      def button_do_stuff(self):
+          batch = self.env['queue.job.batch'].get_new_batch('Group')
+          for i in range(1, 100):
+              self.env['my.model'].with_context(
+                  job_batch=batch
+              ).with_delay().my_method('a', k=i)
```

Comment:
re-add `batch.enqueue()` here

Reply:
Added, it was on the DESCRIPTION.rst, Only the oca-gen-addon-readme command was not executed

Reply:
Ah sorry about that.